docs/libraries/testing-support-mock-console-app.md

```
@@ -6,42 +6,89 @@
+The application is packed **without a native apphost**, so a single package behaves identically on Windows, Linux and macOS. There is no `.exe` — launch it through the `dotnet` muxer. Because `dotnet <app>.dll` runs the application inside the `dotnet` process itself, the `Process` handle you get back is the stub's own process, so process inspection and manipulation (affinity, priority, exit codes) behave as expected.
 
 ## Installation
 
 ```shell
 dotnet add package Ploch.TestingSupport.MockConsoleApp
 ```
 
-After installation, the `Ploch.TestingSupport.MockConsoleApp.exe` (or the platform-equivalent binary) is available in the package's tools directory and can be referenced in test code by path.
+The package payload lives under `tools/<tfm>/` and carries an MSBuild targets file that is imported automatically. On build, the stub is staged into your project's output directory at `MockConsoleApp/`:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)